### PR TITLE
fix(pip_requirements): handle whitespace around ==

### PR DESCRIPTION
**Pull request by jose-ws**
## Changes:

Spaces around a double equals (==) used to confuse renovate into thinking the dependency needed pinning. This PR fixes that.

## Context:

Closes Issue: #8257

## Documentation (please check one with an [x])

- [ ] I have updated the documentation, or
- [x ] No documentation update is required

## How I've tested my work (please tick one)

I have verified these changes via:

- [ ] Code inspection only, or
- [ ] Newly added unit tests, or
- [ ] No new tests but ran on a real repository, or
- [x ] Both unit tests + ran on a real repository

### CODE REVIEW

File: lib/manager/pip_requirements/extract.spec.ts [hunk 1]
```diff
@@ -9,6 +9,7 @@ const requirements4 = loadFixture('requirements4.txt');
 const requirements5 = loadFixture('requirements5.txt');
 const requirements6 = loadFixture('requirements6.txt');
 const requirements7 = loadFixture('requirements7.txt');
+const requirements8 = loadFixture('requirements8.txt');
```
Comment (viceice) [inline lib/manager/pip_requirements/extract.spec.ts:12]: Can you please simply update one of the existing tests by adding some spaces.

File: lib/util/git/__snapshots__/index.spec.ts.snap [hunk 2]
```diff
@@ -6,6 +6,15 @@ Array [
...
 
+exports[`util/git/index getBranchList() should return all branches 1`] = `
+Array [
+  ".gitmodules",
+  "file_to_delete",
+  "master_file",
+  "past_file",
+]
+`;
+
```
Comment (viceice) [inline lib/util/git/__snapshots__/index.spec.ts.snap:17]: This looks wrong to me

**Comment by viceice** [comment]
Please sign the cla

File: lib/util/git/__snapshots__/index.spec.ts.snap (hunk 2 above)
Comment (rarkins) [inline lib/util/git/__snapshots__/index.spec.ts.snap:17]:
```suggestion
```

**Review by rarkins — APPROVED** [review]

**Comment by HonkingGoose** [comment]
@jose-ws Congratulations on getting your first PR landed! 🚀  🎉 ✨ 

A tip: `Closes #9910` would have closed the linked issue after your PR got merged. You were very close to getting it right. 😉 
So you don't need to use the `issue` word in the sequence. 😉 

The GitHub docs for this feature can be found here: https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword

I've manually closed the linked issue, so you do not need to worry.

**Pull request merged**